core/src/main/scala/tech/beshu/ror/boot/ReadonlyRest.scala

```diff
@@ -94,7 +94,7 @@ trait ReadonlyRest extends Logging {
           case LoadEsConfigError.FileNotFound(file) =>
             StartingFailure(s"Cannot find elasticsearch config file: [${file.pathAsString}]")
           case LoadEsConfigError.MalformedContent(file, msg) =>
-            StartingFailure(s"Elasticsearch config file is malformed: [${file.pathAsString}], $msg")
+            StartingFailure(s"Config file is malformed: [${file.pathAsString}], $msg")
```

Comment:
We refer to "settings" rather than "config" in our docs, let's stick to one.

Reply:
done